editors/atril.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -uo pipefail
```

Comment:
Please consider moving security-connected changes to another PR if it's possible

Reply:
I think it is impossible now

Reply:
It is possible, delete the change in this PR by creating new commit, create another branch and introduce the change from the branch (either cherry pick commit which you want or create from scracth)

You are doing all PRs from `master` branch of your fork, it is better to create new branch from your master for each pr you introduce.

Reply:
I moved it to another PR